connectivity/FEATURE_BLE/source/generic/GapImpl.cpp

```diff
@@ -2985,7 +3278,24 @@ void Gap::on_remote_connection_parameter(
 #if BLE_ROLE_OBSERVER
 ble_error_t Gap::setScanParameters(const ScanParameters &params)
 {
+    tr_info("Set scan parameters - "
+            "own_address_type=%s, "
+            "scanning_filter_policy=%s, "
+            "phys=%d ,"
+            "phy_1m_configuration:[interval=%lums,window=%lums,active_scanning=%s] ,"
```

Comment:
```suggestion
            "phy_1m_configuration:[interval=%lums,window=%lums,active_scanning=%s], "
```

Reply:
Thanks. I should also use the `PRIu32` macro here instead of `%lu`.

Reply:
Actually, there are a good few `%lu` lingering in the PR.

Reply:
Some changes from 0a86ce1 were lost somehow.